src/Sylius/Bundle/ShopBundle/Resources/config/grids/product.yml

```diff
@@ -15,7 +15,6 @@ sylius_grid:
                             taxon: "expr:notFoundOnNull(service('sylius.repository.taxon').findOneBySlug($slug, service('sylius.context.locale').getLocaleCode()))"
                             locale: "expr:service('sylius.context.locale').getLocaleCode()"
                             sorting: "expr:service('request_stack').getCurrentRequest().get('sorting', [])"
-                            includeAllDescendants: "expr:parameter('sylius_shop.product_grid.include_all_descendants')"
```

Comment:
At first glance, it should not be removed, we want to have this parameter set here.